src/App.js

```diff
@@ -1,12 +1,164 @@
+                                {/*<button onClick={() => setBanaanAmount (banaanAmount+1)}>+</button>*/}
+
+                                <h2>🍓 Aardbeien</h2>
+
+                                <Fruitcounter
+                                    fruit={aardbeiAmount}
+                                    setFruit={setAardbeiAmount}
+                                    fruitId='aardbeiCounter'
+                                ></Fruitcounter>
+
+
+                                {/*<button disabled={aardbeiAmount === 0} onClick={() => setAardbeiAmount (aardbeiAmount-1)}>-</button>*/}
+                                {/*<label htmlFor="aardbeienCounter"/>*/}
+                                {/*<input type='number' value={aardbeiAmount} name="aardbeienAmount" readOnly='value' id="aardbeienCounter"/>*/}
+                                {/*<button onClick={() => setAardbeiAmount (aardbeiAmount+1)}>+</button>*/}
+
+                                <h2>🍏 Appels</h2>
+
+                                <Fruitcounter
+                                    fruit={appelAmount}
+                                    setFruit={setAppelAmount}
+                                    fruitId='appelCounter'
+                                ></Fruitcounter>
+
+                                {/*<button disabled={appelAmount === 0} onClick={() => setAppelAmount (appelAmount-1)}>-</button>*/}
+                                {/*<label htmlFor="appelCounter"/>*/}
+                                {/*<input type='number' value={appelAmount} name="appelAmount" readOnly='value' id="appelCounter"/>*/}
+                                {/*<button onClick={() => setAppelAmount (appelAmount+1)}>+</button>*/}
+
+                                <h2>🥝 Kiwi's</h2>
+
+                                <Fruitcounter
+                                    fruit={kiwiAmount}
+                                    setFruit={setKiwiAmount}
+                                    fruitId='kiwiCounter'
+                                ></Fruitcounter>
+
+                                {/*<button disabled={kiwiAmount === 0} onClick={() => setKiwiAmount (kiwiAmount-1)}>-</button>*/}
+                                {/*<label htmlFor="kiwiCounter"/>*/}
+                                {/*<input type='number' value={kiwiAmount} name="kiwiAmount" readOnly='value' id="kiwiCounter"/>*/}
+                                {/*<button onClick={() => setKiwiAmount (kiwiAmount+1)}>+</button>*/}
+
+                                <div>
+                                    {/*<button onClick={resetFruitAmount}> Reset </button>*/}
+                                    <Button
+                                        onClick={resetFruitAmount}
+                                        type="button"
+                                        >
+                                            Reset
+                                    </Button>
+
+                                </div>
```

Comment:
Deze div is niet perse nodig
```suggestion
                                <Button
                                    onClick={resetFruitAmount}
                                    type="button"
                                >
                                    Reset
                                </Button>
```